aviary/modules/processor.py

```diff
@@ -112,16 +112,32 @@ def __init__(self,
             self.refinery_max_iterations = args.refinery_max_iterations
             self.refinery_max_retries = args.refinery_max_retries
             self.skip_abundances = args.skip_abundances
+            self.skip_taxonomy = args.skip_taxonomy
+            self.skip_singlem = args.skip_singlem
+            if args.binning_only:
+                self.skip_abundances = True
+                self.skip_taxonomy = True
+                self.skip_singlem = True
+            self.binning_only = args.binning_only
+
+            self.skip_binners = ["maxbin2", "concoct"]
+            if args.extra_binners:
+                for binner in args.extra_binners:
+                    binner = binner.lower()   
+                    if binner == "maxbin" or binner == "maxbin2":
+                        self.skip_binners.remove("maxbin2")
+                    elif binner == "concoct":
+                        self.skip_binners.remove("concoct")
+                    else:
+                        logging.warning(f"Unknown extra binner {binner} specified. Skipping...")

```

Comment:
Could error here instead. @wwood what do you think?
Shouldn't happen anyway due to the argparse choices.